devscripts/make_changelog.py

```diff
@@ -260,7 +260,7 @@ class CommitRange:
     AUTHOR_INDICATOR_RE = re.compile(r'Authored by:? ', re.IGNORECASE)
     MESSAGE_RE = re.compile(r'''
         (?:\[(?P<prefix>[^\]]+)\]\ )?
-        (?:(?P<sub_details>`?[^:`]+`?): )?
+        (?:(?P<sub_details>`?[^\n:`]+`?): )?
```

Comment:
```suggestion
        (?:(?P<sub_details>`?[\w.-]+`?): )?
```
This allows us to only use `\w`, `-` and `.`, but i think we should do that anyways.